src/commands/config.ts

```diff
@@ -0,0 +1,9 @@
+import { Argv } from "yargs";
```

Comment:
There is a problem with this function, it doesn't explain what method is if the argument is missing

![image](https://user-images.githubusercontent.com/503185/107472021-263b5600-6ba9-11eb-9a04-77dd0eed4270.png)

You can run the wrap function with no arguments to compare